src/interest.rs

```diff
@@ -53,6 +55,10 @@ impl Interest {
     #[cfg(target_os = "freebsd")]
     pub const LIO: Interest = Interest(unsafe { NonZeroU8::new_unchecked(LIO) });
 
+    /// Returns a `Interest` set representing LIO completion interests.
+    #[cfg(target_os = "macos")]
```

Comment:
Can you also add `ios`, same for the other `cfg`s.

Reply:
I will, just be aware that I can't test on ios, only macos